.github/workflows/docker-publish.yml

```
@@ -2,9 +2,9 @@ name: Docker
 
 on:
   push:
-    branches: [main]
+    branches: [ main ]
```

YAMLlint (1.37.1)

[error] too many spaces inside brackets

(brackets)